x-pack/test/security_solution_api_integration/test_suites/detections_response/basic_essentials_license/rule_management/coverage_overview.ts

```diff
@@ -0,0 +1,585 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import {
+  CoverageOverviewRuleActivity,
+  CoverageOverviewRuleSource,
+  RULE_MANAGEMENT_COVERAGE_OVERVIEW_URL,
+  ThreatArray,
+} from '@kbn/security-solution-plugin/common/api/detection_engine';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+import {
+  createPrebuiltRuleAssetSavedObjects,
+  createRuleAssetSavedObject,
+  createRule,
+  deleteAllRules,
+  installPrebuiltRulesAndTimelines,
+  installPrebuiltRules,
+  getCustomQueryRuleParams,
+} from '../../utils';
+import { getCoverageOverview } from '../../utils/rules/get_coverage_overview';
+
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const log = getService('log');
+  const es = getService('es');
+  const alertingApi = getService('alertingApi');
+
+  describe('@serverless @ess coverage_overview', () => {
+    beforeEach(async () => {
+      await deleteAllRules(supertest, log);
+    });
+
+    it('does NOT error when there are no security rules', async () => {
+      // Creates a non-security type rule
+      await alertingApi.createRule({
+        consumer: 'security',
+        name: 'Threshold rule',
+        ruleTypeId: 'observability.rules.custom_threshold',
+        params: {
+          criteria: [],
+          searchConfiguration: {
+            query: {
+              query: '',
+              language: 'kuery',
+            },
+            index: 'data-view-id',
+          },
+        },
+      });
+      const rule1 = await createRule(
+        supertest,
+        log,
+        getCustomQueryRuleParams({ threat: generateThreatArray(1) })
+      );
+
+      const body = await getCoverageOverview(supertest);
+
+      expect(body).to.eql({
+        coverage: {
+          T001: [rule1.id],
+          TA001: [rule1.id],
+          'T001.001': [rule1.id],
+        },
+        unmapped_rule_ids: [],
+        rules_data: {
+          [rule1.id]: {
+            activity: 'disabled',
+            name: 'Custom query rule',
+          },
+        },
+      });
+    });
+
+    describe('without filters', () => {
```

Comment:
What about cases when two or more rules are mapped to the same technique via different tactics? Under these techniques, there can also be different or the same sub-techniques. Should we cover all these cases here?

Reply:
I don't think those should be covered here as we do all the logic for mapping those on the front end, it will always come back from the backend the same. They would need to be unit tests (some which we already have) or cypress tests